include/core/jbackend.h

```diff
@@ -429,6 +436,7 @@ gboolean j_backend_db_delete (JBackend*, gpointer, gchar const*, bson_t const*,
 gboolean j_backend_db_query (JBackend*, gpointer, gchar const*, bson_t const*, gpointer*, GError**);
 gboolean j_backend_db_iterate (JBackend*, gpointer, bson_t*, GError**);
 
+gboolean j_backend_reset (JBackend* backend, gpointer, GError**);
```

Comment:
See above for naming.

Reply:
the intention was to have one reset function for all backend types